diff -ur swish-e.orig/src/config.h swish-e/src/config.h
--- swish-e.orig/src/config.h	2002-12-04 18:18:52.000000000 -0500
+++ swish-e/src/config.h	2002-12-04 18:07:15.000000000 -0500
@@ -314,7 +314,7 @@
 ** in getrank; don't just go changing these values!
 */
 
-#define SPIDERDIRECTORY "./"
+#define SPIDERDIRECTORY "/usr/lib/swish-e/"
 
 #define SWAP_LOC_DEFAULT 0
 
Only in swish-e: swish-e-2.2.2.diff
